portfolio/src/main/java/com/google/sps/servlets/DeleteServerlet.java

```diff
@@ -0,0 +1,33 @@
+package com.google.sps.servlets;
```

Comment:
Remove the delete comment parts that are not relevant to this PR?

Reply:
I think they should be removed after I merge the previous PR. Sorry, I should have made this branch from the most recent commit on master instead.